src/task/ReceiveTURNCredentialsTask.ts

```diff
@@ -62,6 +63,12 @@ export default class ReceiveTURNCredentialsTask extends BaseTask {
       try {
         const responseBody = await fetch(Versioning.urlWithVersion(this.url), options);
         this.context.logger.info(`received TURN credentials`);
+        if (responseBody.status && responseBody.status === 403) {
+          let error = new Error(
+            `Forbidden to fetch TURN Credentials due to the meeting status code: ${MeetingSessionStatusCode.ForbiddenToFetchTrunCredentials}`
```

Comment:
This string is the only place where this error code is used, so why do we need it?

Reply:
Do you suggest using an existing error code for this case?

Reply:
As far as I can tell, the error code is not set or returned anywhere. If no one can actually consume this error code (other than by parsing it out of a string—which could change), then what is the purpose of having an error code?